src/components/Hand.js

```diff
@@ -28,25 +26,58 @@ const CardImg = styled.img`
   position: relative;
   margin-top: ${({ selected }) => (selected ? '-350px' : '-250px')};
   cursor: ${({ onClick }) => (onClick ? 'pointer' : null)};
-  top: 125px;
+  width: 100px;
+  top: 50px;
+  ${({ primary }) => {
+    return primary ? 'transform: scale(1.5);' : '';
+  }};
+
   &:not(:first-child) {
-    margin-left: -195px;
+    ${({ primary }) => {
+      return primary ? 'margin-left: -75px;' : 'margin-left: -92px;';
+    }};
+  }
+
+  @media (min-width: ${({ theme }) => {
+      return theme.screen.smMin;
+    }}) {
```

Comment:
For the codenames layout, I ended up scaling based on both the width and height of the screen (if the screen was too small in either dimension, I scaled down the graphics). I'm not sure how playable this is going to end up being on a mobile phone, but we'll have to consider both horizontal and vertical layouts perhaps.

Let's focus on a playable game on a PC first I guess.

Reply:
I could view it with the small screens - plus the dragging would be cool with the mobile

Reply:
also I added a change for this with the screen ratio so you can switch landscape/portrait, I'm not used to using the vert queries since css heights are always so weird - I just look at the actual window changes and handle it with js